documentation/src/main/asciidoc/ch12.asciidoc

```diff
@@ -132,9 +132,9 @@ _hibernate.validator.allow_parallel_method_parameter_constraint_ in _validation.
 ----
 <?xml version="1.0" encoding="UTF-8"?>
 <validation-config
-        xmlns="https://xmlns.jakarta.ee/xml/ns/validation/configuration"
+        xmlns="https://jakarta.ee/xml/ns/validation/configuration"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="https://xmlns.jakarta.ee/xml/ns/validation/configuration validation-configuration-3.0.xsd"
+        xsi:schemaLocation="https://jakarta.ee/xml/ns/validation/configuration validation-configuration-3.0.xsd"
```

Comment:
The xsi:schemaLocation here is not correct as the "validation-configuration-3.0.xsd" relative URL would not resolve correctly as the actual location on the jakarta.ee website is https://jakarta.ee/xml/ns/validation/validation-configuration-3.0.xsd.

The validation-configuration-3.0.xsd documentation example uses the correct form:

            <validation-config
                xmlns="https://jakarta.ee/xml/ns/validation/configuration"
                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                xsi:schemaLocation="
                    https://jakarta.ee/xml/ns/validation/configuration
                    https://jakarta.ee/xml/ns/validation/validation-configuration-3.0.xsd"
                version="3.0">

Reply:
Hum, hum. HV should never actually fetch the XSD from there, but rather use its own local copy of the file. I.e. the value here shouldn't make any actual difference; it should be more taken as a visual cue than as a resolveable resource. I don't think we ever had the full path there in examples.

Reply:
Sure, HV does not need it, but if people are configuring an IDE or other xml aware parser it helps to have the actual resolvable location. For example, in Intellij if I use the full form of the schemaLocation and it shows up as red I can simply use the resolve external location hint to register the schema file with the namespace.

Reply:
Fixed.